docs/tech/aca.md

```diff
@@ -0,0 +1,119 @@
+# Attribute Certificate Authority
+
+To support Attribute Based Access Control (ABAC) the user has to pass a set of attributes during TCert creation, these attributes will be used during transaction execution or query to determine whether the user can or cannot execute a specific chaincode. A mechanism to validate the ownership of attributes is required in order to prove if the attributes passed by the user are correct. The Attribute Certificate Authority (ACA) has the responsibility of validating attributes and in response it returns an Attribute Certificate (ACert) with the valid attribute values.
```

Comment:
In 75e1564 I have redefined the structure of this document. Can you check if you are agree with this new version?